RNDateTimePicker.podspec

```diff
@@ -12,8 +14,28 @@ Pod::Spec.new do |s|
   s.homepage     = package['homepage']
   s.platform     = :ios, "11.0"
   s.source       = { :git => "https://github.com/react-native-community/datetimepicker", :tag => "v#{s.version}" }
-  s.source_files = "ios/*.{h,m}"
+  s.source_files = "ios/**/*.{h,m,mm}"
   s.requires_arc = true
 
-  s.dependency "React-Core"
+  if fabric_enabled
+    folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
```

Comment:
Do you have to exclude any of the legacy files here?

Reply:
no